style.css

```diff
@@ -0,0 +1,115 @@
+
+body {
+    text-align: center;
+    width: 100%;
+    line-height: 3em;
+}
+
+#keyboard {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    border: 2px solid #6292c2;;
+    padding: 5px;
+    max-width: 900px;
+    margin: 0 auto;
+    text-align: center;
+    position: relative;
+}
+
+#keyboard::after {
+    content: "";
+    display: block;
+    clear: both;
+}
+
+.k-key {
+    width: 50px;
+    height: 50px;
+    border: 1px solid black;
+    margin: 3px;
+    padding: 3px;
+    cursor: pointer;
+    background: #6292c2;
+    border: 2px solid #eee;
+    -webkit-border-radius: 50px;
+    -moz-border-radius: 50px;
+    border-radius: 15px;
+    /* transition: All 1s ease; */
+    text-decoration: none;
+  outline: none;
+  display: inline-block;
+  color: white;
+  border-radius: 10px;
+  font-family: 'Montserrat', sans-serif;
+  text-transform: uppercase;
+  letter-spacing: 2px;
+  background-image: linear-gradient(to right, #9EEFE1 0%, #4830F0 51%, #9EEFE1 100%);
+  background-size: 200% auto;
+  box-shadow: 0 0 20px rgba(0,0,0,.1);
+  transition: .5s;
+}
+
+.error {
+    color: tomato;
+}
+
+.k-key:hover {
+    opacity: 80%;
+    background-position: right center;
+}
+
+.k-key:active {
+    border-radius: 50%;
+    transition: .1s;
+    background-position: right center;
+}
+
+.clearfix {
+    display: block;
+    width: 100%;
+    clear: both;
+}
+
+.active {
+    background: orange;
+}
+
+textarea {
+    width: 400px;
+    height: 100px;
+    border-color: tomato;
+    font-size: 16px;
+}
+
+.language {
+    width: 900px;
+    height: 30px;
+    margin: 0 auto;
+    margin-bottom: 30px;
+    line-height: 1em;
+}
+
+.changeLanguage {
+    position: static;
```

Comment:
static; - default value of position.
Don't need to set static, because this element has already position: static;

Reply:
Of course  I knew it) I don't know Why i did it) It won't happen again

Reply:
I've just corrected it.